e2e/src/page-objects/pages/send.page.ts

```diff
@@ -9,6 +9,8 @@ export class SendPage extends Page {
   amountInput = createPageElement(SendFormSelectors.amountInput);
   recipientInput = createPageElement(SendFormSelectors.recipientInput);
   sendButton = createPageElement(SendFormSelectors.sendButton);
+  contactItemButton = createPageElement(SendFormSelectors.contactItemButton);
+  contactHashValue = createPageElement(SendFormSelectors.contactHashValue);
```

Comment:
These elements aren't used anywhere. Is it just as planned?

Reply:
They are in the page object for not forgetting in the future. I think they should be left here